R/AllGenerics.R

```diff
@@ -122,6 +122,10 @@ setGeneric("groupClass", function (x) standardGeneric("groupClass"))
 setGeneric("entityClass", function (x) standardGeneric("entityClass"))
 setGeneric("entitiesInitializer", function (x) standardGeneric("entitiesInitializer"))
 setGeneric("weightVariables", function (x) standardGeneric("weightVariables"))
+setGeneric("modifyWeightVariables", function (x, vars, type) standardGeneric("modifyWeightVariables"))
```

Comment:
Since you've turned this into a function, you don't need this generic definition.